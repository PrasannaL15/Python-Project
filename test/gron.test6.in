{
  "empty_object": {},
  "nested_empty_object": {
    "inner_empty_object": {}
  }
}
