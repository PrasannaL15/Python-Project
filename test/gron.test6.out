json = {}
json.empty_object = {}
json.nested_empty_object = {}
json.nested_empty_object.inner_empty_object = {}

